### PR TITLE
Bug 1767990 - Expose `last_activity_ts` in the /rest/user endpoint

### PR DESCRIPTION
**Pull request by suhaibmujahid**
This PR is totally untested.

### CODE REVIEW

File: Bugzilla/WebService/User.pm [hunk 1]
```diff
@@ -298,14 +298,15 @@ sub get {
...
-      last_seen_date => $self->type('dateTime',  $user->last_seen_date),
+      id                 => $self->type('int',      $user->id),
+      real_name          => $self->type('string',   $user->name),
+      nick               => $self->type('string',   $user->nick),
+      name               => $self->type('email',    $user->login),
+      email              => $self->type('email',    $user->email),
+      can_login          => $self->type('boolean',  $user->is_enabled ? 1 : 0),
+      iam_username       => $self->type('string',   $user->iam_username),
+      last_seen_date     => $self->type('dateTime', $user->last_seen_date),
+      last_activity_date => $self->type('dateTime', $user->last_activity_date),
```
Comment (dklawren) [inline Bugzilla/WebService/User.pm:309]: `last_activity_time => $self->type('dateTime', $user->last_activity_time),`

File: docs/en/rst/api/core/v1/user.rst [hunk 2]
```diff
@@ -346,40 +346,41 @@ include_disabled  boolean  By default, when using the ``match`` parameter,
...
-saved_searches     array      User's saved searches, each having the following
+saved_searches      array     User's saved searches, each having the following
                               Search object items described below.
-saved_reports      array      User's saved reports, each having the following
+saved_reports       array     User's saved reports, each having the following
                               Search object items described below.
-last_seen_date     datetime   The time when the user last loaded any page.
-=================  =========  =====================================================
+last_seen_date      datetime  The time when the user last loaded any page.
+last_activity_date  datetime  The time when the user last made a change to a bug.
```
Comment (dklawren) [inline docs/en/rst/api/core/v1/user.rst:382]: `last_activity_time  datetime  The time when the user last made a change to a bug.`

File: Bugzilla/User.pm [hunk 3]
```diff
@@ -79,7 +79,8 @@ sub DB_COLUMNS {
     'profiles.disable_mail',
     'profiles.extern_id',
     'profiles.is_enabled',
-    $dbh->sql_date_format('last_seen_date', '%Y-%m-%d') . ' AS last_seen_date',
+    $dbh->sql_date_format('last_seen_date',   '%Y-%m-%d') . ' AS last_seen_date',
+    $dbh->sql_date_format('last_activity_ts', '%Y-%m-%d') . ' AS last_activity_date',
```
[the remark this answers is not in the document]
Comment (suhaibmujahid) [reply Bugzilla/User.pm]: Done in d7c0a9a80d3ff73b460c2ce9c944778fbd8a3887

File: Bugzilla/User.pm [hunk 4]
```diff
@@ -666,6 +667,7 @@ sub showmybugslink { $_[0]->{showmybugslink}; }
 sub email_disabled { $_[0]->{disable_mail} || !$_[0]->{is_enabled}; }
 sub email_enabled  { !$_[0]->email_disabled; }
 sub last_seen_date { $_[0]->{last_seen_date}; }
+sub last_activity_date       { $_[0]->{last_activity_date}; }
```
[the remark this answers is not in the document]
Comment (suhaibmujahid) [reply Bugzilla/User.pm]: Done in d7c0a9a80d3ff73b460c2ce9c944778fbd8a3887

File: Bugzilla/WebService/User.pm (hunk 1 above)
Comment (suhaibmujahid) [reply Bugzilla/WebService/User.pm]: Done in d7c0a9a80d3ff73b460c2ce9c944778fbd8a3887

File: docs/en/rst/api/core/v1/user.rst (hunk 2 above)
Comment (suhaibmujahid) [reply docs/en/rst/api/core/v1/user.rst]: Done in d7c0a9a80d3ff73b460c2ce9c944778fbd8a3887

**Review by dklawren — APPROVED** [review]
LGTM. r=dkl

**Pull request merged**